vue-components/stories/Dialog.stories.ts

```diff
@@ -0,0 +1,125 @@
+import Dialog from '@/components/Dialog';
+import Button from '@/components/Button';
+import { Component } from 'vue';
+
+export default {
+	component: Dialog,
+	title: '/Dialog',
+};
+
+export function basic( args: { title: string, actions: string, dismissButton: boolean, visible: boolean }): Component {
+	return {
+		components: { Button, Dialog },
+		props: Object.keys( args ),
+		template: `
+			<div>
+				<Button 
+					@click.native="$refs.simple.show()" 
+					variant="primary" type="progressive"
+				>
+					Open Basic Dialog
+				</Button>
+				<Dialog
+					:title="title"
+					ref="simple"
+					:actions="actions"
+					style="transform: translate(50%, 50%)"
+					@action="(_, dialog) => dialog.hide()"
```

Comment:
I took this from the MMF. should this be the generic action?

Reply:
I am not sure if I understand the question? If you mean to say should the dialog just close on any button click then yes, it should

Reply:
right now there is only one action, yes, so both buttons close the window. is this desired = should i just leave it like this? :)